public/css/components.css

```
@@ -2590,3 +2584,244 @@ html[data-motion="reduced"] .help-tip[data-show="true"] { transform: none; }
+  outline: none;
+  box-shadow: var(--accent-ring, 0 0 0 3px rgba(144,103,198,0.14));
+}
+.launch-pad-submit[disabled] {
+  opacity: 0.4;
+  cursor: not-allowed;
+}
+.launch-pad-submit svg {
+  width: 16px;
+  height: 16px;
+  fill: none;
+  stroke: currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)